<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIMainWindow</class>
 <widget class="QMainWindow" name="UIMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MO2IL</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="UIToolBar" name="toolbar" native="true"/>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>MO2</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="UIMO2View" name="mo2View" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Mods</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="UIModListView" name="modListView" native="true"/>
          <widget class="UIModFileView" name="modFileView" native="true"/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>競合ファイルチェック</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="UIOverrideCheck" name="widget_2" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>ファイル検索</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_8" stretch="0">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="UIMainSearch" name="widget" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>713</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFallout4">
    <property name="title">
     <string>ファイル</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionDebug"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>ツール</string>
    </property>
    <addaction name="actionaa"/>
   </widget>
   <addaction name="menuFallout4"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>ログ</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="HLogView" name="logWidget" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>開く</string>
   </property>
  </action>
  <action name="actionbbb">
   <property name="text">
    <string>bbb</string>
   </property>
  </action>
  <action name="actionDebug">
   <property name="text">
    <string>Debug</string>
   </property>
  </action>
  <action name="actionaa">
   <property name="text">
    <string>[SCTX - Script Source]をファイルに書き出す </string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HLogView</class>
   <extends>QWidget</extends>
   <header>HLogView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UIModListView</class>
   <extends>QWidget</extends>
   <header>UIModListView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UIToolBar</class>
   <extends>QWidget</extends>
   <header>UIToolBar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UIModFileView</class>
   <extends>QWidget</extends>
   <header>UIModFileView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UIMO2View</class>
   <extends>QWidget</extends>
   <header>UIMO2View.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UIOverrideCheck</class>
   <extends>QWidget</extends>
   <header>UIOverrideCheck.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UIMainSearch</class>
   <extends>QWidget</extends>
   <header>UIMainSearch.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
